packages/eslint-config-eventbrite/rules/best-practices.js

```diff
@@ -4,5 +4,10 @@ module.exports = {
         // disallow modifying properties of parameters
         // http://eslint.org/docs/rules/no-param-reassign
         'no-param-reassign': ['error', {props: true}],
+
+        // allow `this` keywords outside of classes or class-like objects
+        // so that `babel/no-invalid-this` can disallow (see es6 rules)
+        // http://eslint.org/docs/rules/no-invalid-this
+        'no-invalid-this': 'off',
```

Comment:
This makes sense due to the comment at the top about overriding legacy, but not sure I understand.

In legacy we say no because of backbone (etc), but best practice is that we want it off, but we still limit it in es6?

I think putting this next to the babel override would make more sense in my head, but not sure the logic translates to the config.

Reply:
Yean I was trying to figure out the best place to put it. Since it's disallowed in the legacy best-practices, I figured it'd make sense to turn it off in the base best-practices. But like you said it's weird w/o the context of what's happening with the babel version. But I believe the babel one should be in the es6 one (or maybe esnext).

I think no matter where it's placed, the overriding will happen so it should be ok. I'll move it all to esnext for clarity

Reply:
Also just realized that I turned `babel/no-invalid-this` `'off'` instead of `'error'`!

Reply:
Yeah was really breaking my mental model.